src/components/Field/Field.tsx

```diff
@@ -36,11 +36,14 @@ export default class Field extends Component<FieldProps, FieldState> {
           selectedImage: this.state.selectedAsset,
         },
       })
-      .then((selectedAsset) =>
-        this.setState({ selectedAsset }, () =>
+      .then((selectedAsset) => {
+        if (!selectedAsset) {
+          return;
+        }
+        return this.setState({ selectedAsset }, () =>
```

Comment:
Is it necessary to `return` here?

Reply:
No, this was a quick and dirty way of doing validating my hypothesis. I cleaned this up in a new commit.